### PR TITLE
feat: TRP docs

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Review by krogla — APPROVED** [review]
lgtm

**Review by vgorkavenko — DISMISSED** [review]

File: docs/contracts/trp-vesting-escrow.md [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+    - [VotingAdapter](https://goerli.etherscan.io/address/0x4d5C16778B7632e7B0C24fF102072247b9cf34c2)
+- [Detailed contracts spec](https://hackmd.io/FZEW9MM7QwmUehFtKDmqnw)
+
+[Token Reward Program (TRP)](https://research.lido.fi/t/lidodao-token-rewards-plan-trp/3364) escrow contracts allow transparent on-chain distribution and vesting of the token rewards for the Lido DAO contributors.
+
+## VestingEscrowFactory
+
+### Public variables
+
+- `voting_adapter: address` - address of the VotingAdapter used in the vestings
```
Comment (TheDZhon) [inline docs/contracts/trp-vesting-escrow.md:20]: `VotingAdapter`

**Review by TheDZhon — APPROVED** [review]
It's totally cool, thank you!
Left some suggestions.

:warning:  Please don't merge before the public announcement.

**Review by zuzueeka — APPROVED** [review]

File: docs/deployed-contracts/index.md [hunk 2]
```diff
@@ -124,6 +124,7 @@
 - Jumpgate for [Solana Reward Program](https://solscan.io/address/H6Q8UwkZ7G6qmBv4Nke7BtzbHrfZNvFhdQPCmUUHtTy2): [`0xe3224542066d3bbc02bc3d70b641be4bc6f40e36`](https://etherscan.io/address/0xe3224542066d3bbc02bc3d70b641be4bc6f40e36).
 - Tokens recoverer for Manager contracts ([Reward Programs](#reward-programs)): [`0x1bdfFe0EBef3FEAdF2723D3330727D73f538959C`](https://etherscan.io/address/0x1bdfFe0EBef3FEAdF2723D3330727D73f538959C).
 - Jumpgate for [Solana Referral Program](https://solscan.io/address/7KN9sjxsStxsTTPXZBZk6z1cdXfVkyvq1ihAVh94gXcP): [`0x13C6eF8d45aFBCcF15ec0701567cC9fAD2b63CE8`](https://etherscan.io/address/0x13C6eF8d45aFBCcF15ec0701567cC9fAD2b63CE8).
+- TRP [VestingEscrowFactory](https://etherscan.io/address/0xDA1DF6442aFD2EC36aBEa91029794B9b2156ADD0)
```
[the remark this answers is not in the document]
Comment (dgusakov) [reply docs/deployed-contracts/index.md]: Done

**Pull request merged**